index.html

```diff
@@ -24,81 +25,29 @@
                 </a>
             </button>
 
-            <button id="header-btn2">
-                <nav>
-                    <svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
-                        <path fill-rule="evenodd" clip-rule="evenodd"
-                            d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
-                            fill="#6070FF" />
-                    </svg>
-                </nav>
-            </button>
+            <nav class="nav-bar">
+                <div class="nav-bar-frame"><a href="/portfolio/">Portfolio</a></div>
+                <div class="nav-bar-frame"><a href="/about/">About</a></div>
+                <div class="nav-bar-frame"><a href="/contact/">Contact</a></div>
+                <button id="header-btn2">
+                    <nav>
+                      <img src="./icons/Union.png" alt="">
+                    </nav>
+                </button>
+            </nav>
```

Comment:
- [ ] The navigation menus are missing from your desktop design. Kindly check and make sure they are present to match the actual design. 
- [ ] The headline part of your design seems to be pushed towards the left side with the background covering that part alone leaving a blank empty space to the right as shown below. Kindly check and fix it to match the design. 

### Your design 

![Screenshot from 2023-03-03 19-17-21](https://user-images.githubusercontent.com/65233860/222809231-cf880942-1811-4416-941d-407d2fe992a4.png)



### Figma design


![Screenshot from 2023-03-03 19-19-15](https://user-images.githubusercontent.com/65233860/222809246-6e34836a-80a9-4ffc-af02-1dee9624825a.png)